packages/kit/types/ambient.d.ts

```diff
@@ -2,41 +2,26 @@
  * It's possible to tell SvelteKit how to type objects inside your app by declaring the `App` namespace. By default, a new project will have a file called `src/app.d.ts` containing the following:
  *
  * ```ts
- * /// <reference types="@sveltejs/kit" />
- *
- * declare namespace App {
- * 	interface Error {}
- * 	interface Locals {}
- * 	interface PageData {}
- * 	interface Platform {}
- * }
- * ```
- *
- * By populating these interfaces, you will gain type safety when using `event.locals`, `event.platform`, and `data` from `load` functions.
- *
- * Note that since it's an ambient declaration file, you have to be careful when using `import` statements. Once you add an `import`
- * at the top level, the declaration file is no longer considered ambient and you lose access to these typings in other files.
- * To avoid this, either use the `import(...)` function:
- *
- * ```ts
- * interface Locals {
- * 	user: import('$lib/types').User;
- * }
- * ```
- * Or wrap the namespace with `declare global`:
- * ```ts
- * import { User } from '$lib/types';
- *
  * declare global {
  * 	namespace App {
- * 		interface Locals {
- * 			user: User;
- * 		}
- * 		// ...
+ * 		// interface Error {}
+ * 		// interface Locals {}
+ * 		// interface PageData {}
+ * 		// interface Platform {}
  * 	}
  * }
+ *
+ * export default undefined;
  * ```
  *
+ * By populating these interfaces, you will gain type safety when using `event.locals`, `event.platform`, and `data` from `load` functions.
+ *
+ * Note that since it's a declaration file, you have to be careful when using `import` statements. Once you add an `import`
+ * at the top level, the declaration file is no longer considered ambient, at which point you need to wrap `App` inside `declare global`
+ * to still be available throughout the app. Opposite to that is the behavior when there's no `import`, at which point having `declare global`
+ * _prevents_ the `App` types from being available.
+ * To safe you the headache of converting between the two this we provide a dummy `export` to force the file being a module.
+ * Do not remove it unless you have an `import` statement at the top level.
```

Comment:
```suggestion
```

Reply:
In fact the text is a little confusing at the moment — it basically reads

> ```ts
> declare global {...}
> ```
>
> If you add an import, you'll need to change this so it says `declare global`

Hopefully only a handful of people who started projects before this change will get tripped up by imports — everyone else has nothing to worry about